### ISSUE TITLE
DateTime.Now Throw Exceptions

### ISSUE THREAD

**Issue by duhaly** [artiomchi/FlexLabs.Upsert#28]
for example:
db.Agreements.Upsert(new Agreement{ 
         Id=1,
         Name="bill",
         CreatedOn= DateTime.Now,
         ModifiedOn = DateTime.Now
}).on(x=>new{ x.Id})
.WhenMatched( x=>new Agreement{   Id=1,
         ModifiedOn = DateTime.Now})
.RunAsync();

**Comment by artiomchi** [comment]
Yup, you're absolutely right! Patch incoming, thanks for spotting this!
